ppr-ui/src/components/common/ButtonsStacked.vue

```diff
@@ -147,9 +148,10 @@ export default defineComponent({
     const cancel = () => {
       emit('cancel', true)
     }
-    const submit = () => {
+    const submit = debounce(() => {
```

Comment:
I think it technically could, you bet.
This will prevent multiple multiple function calls within this set timeframe (ie double clicking or whatever) but it's not directly awaiting an api response, so the function might be invokable after that set timeframe. 

I'm not sure if lodash.debounce has some internal await on the original function being returned however, might have to dig in to there documentation to know, if that was the case it may help, but don't know offhand. 

We could do the latter as well or instead of, it that is the requirement for sure. 
I missed most the context behind the issue but we can discuss at stand up!

Reply:
Just to follow up here, Dima addressed the reason for Debounce in the ticket itself. 

The jist: 
There is a loading state in place that will await the network request, the issue is the UI cycle it takes to trigger that state. 
The implementation of Debounce here will cover the gap between the button press and the loading state triggering (which will await the request response) 

So i guess the answer is Debounce itself wouldn't prevent the resubmission by itself, but it is working in conjunction with that UI loading state. 
So i think this should be A-OK!